docs/api.md

```diff
@@ -713,8 +716,10 @@ Model:
 
   "online": boolean,
 
+  "mentions": [ Message ] // List of messages that mention this user
```

Comment:
Missing comma.

Reply:
We don't have commas in [some?] other models of the API docs; plus it's not particularly standard to anyone but us. I'd like to leave it off if I'm honest.

Reply:
Just committed to make everything more consistent.

Reply:
The problem here is that there are more properties after this one (`authorized`, `email`).